app/forms/nsm/steps/solicitor_declaration_form.rb

```diff
@@ -11,9 +11,9 @@ def persist!
           application.status = :submitted
           application.update_disbursement_positions!
           application.update!(attributes)
-          SubmitToAppStore.perform_later(submission: application)
-          true
         end
+        SubmitToAppStore.perform_later(submission: application)
+        true
```

Comment:
Do we not also want to capture the result of `update!` here then?

Reply:
Ah, not in this case, because `update!` will never return false - the exclamation means succeed or raise an error.